components/button-group/w-button-group.vue

```diff
@@ -18,11 +19,14 @@ const props = defineProps({
 })
 
 provide('outlined', toRef(props, 'outlined'))
+provide('vertical', toRef(props, 'vertical'))
 
-const classes = computed(() => ({
-  ['border border-gray-300']: props.outlined,
-  ['filter drop-shadow-10']: props.raised,
-  [props.vertical ? 'divide-y' : 'divide-x']: true,
-  ['flex-col']: props.vertical
-}))
+const nonOutlinedClass = props.vertical ? ccButtonGroup.nonOutlinedVertical : ccButtonGroup.nonOutlinedHorizontal
```

Comment:
This should be a computed as well 🙂

Reply:
[6201ede](https://github.com/warp-ds/vue/pull/23/commits/6201ede3bafd278bd25dbd9b2383a82a8f79fe0b)